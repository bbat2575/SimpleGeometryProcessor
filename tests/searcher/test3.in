-17.34, 10.66
-17.34, 10.66
32.5, -49.13
32.5, -49.13
7.33, 45.89
7.33, 45.89
31.74, -38.37
31.74, -38.37
22.81, -40.99
22.81, -40.99
-6.29, 37.55
-6.29, 37.55
-42.82, -28.33
-42.82, -28.33
-12.23, -47.58
-12.23, -47.58
19.82, 26.45
19.82, 26.45
48.09, -44.95
48.09, -44.95
