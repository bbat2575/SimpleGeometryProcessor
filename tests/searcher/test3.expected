read 20 points
31.74, -38.37
31.74, -38.37
22.81, -40.99
This is not a triangle
